enhancements/baremetal/an-slo-for-baremetal.md

```diff
@@ -0,0 +1,592 @@
+---
+title: an-slo-for-baremetal
+authors:
+  - "@markmc"
+reviewers:
+  - "@abhinavdahiya"
+  - "@dhellmann"
+  - "@enxebre"
+  - "@eparis"
+  - "@hardys"
+  - "@sadasu"
+  - "@smarterclayton"
+  - "@stbenjam"
+approvers:
+  - TBD
+creation-date: 2020-02-13
+last-updated: 2020-02-21
+status: provisional
+see-also:
+ - https://github.com/markmc/cluster-baremetal-operator
+replaces:
+superseded-by:
+---
+
+# An SLO for baremetal
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+1. ~~How to handle upgrades?~~
+2. Which release should this change target?
+
+## Summary
+
+The `cluster-baremetal-operator` component will provide bare metal
+machine management capabilities needed for the Machine API provider on
```

Comment:
"machine" in this context usually refers to the kubernetes API machine resource.
May be "will provide raw infrastructure management capabilities needed to satisfy the metal3 bare metal Machine API provider requests" would be more accurate here?

Probably good to include a doc glossary: machine (crd), host(crd), instance/infrastructure.